cts/valgrind-pcmk.suppressions

```diff
@@ -301,3 +301,16 @@
    fun:call_init.part.0
    fun:_dl_init
 }
+
+{
+   cmdline preprocessing
+   Memcheck:Leak
+   match-leak-kinds: definite
+   fun:malloc
+   fun:realloc
+   fun:g_realloc
+   obj:*/libglib-*
+   fun:g_ptr_array_add
+   fun:pcmk__cmdline_preproc
```

Comment:
This might be legit. pcmk__cmdline_preproc() allocates the individual strings using strdup() but the callers free it with g_strfreev(), which I'm guessing uses the glib memory management when freeing the individual strings, meaning we should be using g_strdup().

Reply:
Fixed, I think.